src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -194,6 +198,18 @@ protected void updateItem(final TableRow item, final boolean empty) {
       rootItem.setExpanded(true);
    }
 
+   private void collapseExpandTable(){
+      for (int i=0; i<rootItem.getChildren().size(); i++){
```

Comment:
the button should expand all or collapse all. it should not 'stupidly' toggle the current status. would not be to useful.
with your logif currently e.g.  you collapsed one manualy (all others are still expanded). you press the button. now the collapsed one will expand and the others collapse

Reply:
Button now switches between the states "expand all" and "collapse all". 
The problem is now if you manually collapse all then the button will still have the text "collapse". If that happens you have to press the button twice to expand them all again.